libpdb/client_config.go

```diff
@@ -0,0 +1,36 @@
+package libpdb
+
+import (
+	"encoding/json"
+	"github.com/ryscheng/pdb/common"
+	"io/ioutil"
+  "time"
+)
+
+type ClientConfig struct {
+	*common.CommonConfig `json:"-"`
+
+  // How often should Writes be made to the server
+	WriteInterval  time.Duration
```

Comment:
Does it really make sense to factor this out?
These are in fact global parameters. Each client has to be configured with the same values and the server may use these to do things like rate-limiting.